src/auth/authorize.rs

```diff
@@ -304,6 +304,7 @@ fn parse_header_value<Nk: ArrayLength<u8>>(
                 Token::tls_deserialize(
                     &mut URL_SAFE
                         .decode(token_value)
+                        .or_else(|_| STANDARD.decode(token_value))
```

Comment:
this should be [URL_SAFE](https://docs.rs/base64/latest/base64/engine/general_purpose/index.html), as the header is in base64url, not base64 in my understanding